oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -471,7 +471,6 @@ configuration:
     apolloCluster: ${SW_CONFIG_APOLLO_CLUSTER:default}
     apolloEnv: ${SW_CONFIG_APOLLO_ENV:""}
     appId: ${SW_CONFIG_APOLLO_APP_ID:skywalking}
-    period: ${SW_CONFIG_APOLLO_PERIOD:60}
```

Comment:
Can you remove this too in the configuration doc?

Reply:
Ok.